lib/phlex/rails/never.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Phlex::Rails::Never < BasicObject
+	def initialize(&block)
+		@block = block
+	end
+
+	def method_missing(method_name, *, **)
+		@block.call(method_name, *, **)
+	end
+end
```

Comment:
This is so creative!

Reply:
You should see the PR I gave up on. It was popping things off the buffer. 🤣